Salesforce_Advanced/Work_Dot_Com_Command_Center/README.md

```diff
@@ -37,7 +40,7 @@ Parts of the Work.com solution are not automatically configured in step 1.  The
 
 After creating the `Authorization Form Text` record, click into its detail screen. If you followed Salesforce's example, that means clicking `Wellness Authorization Text` in the  `Authorization Form Text` pane. 
 Once you are on the Authorization Form Text detail screen, copy its  `ID` field for use in the Skuid page found below.   It is most easily viewed in the URL of the detail page.  It will be formatted similar to `0cN1700000003KiEAI`.
-For example, in a URL like `https://your-scratch-org-000.lightning.force.com/lightning/r/AuthorizationFormText/0cI3F0000008RM6UAM/view`, the ID would be `0cI3F0000008RM6UAM`.
+For example, in a URL like `https://your-scratch-org-000.lightning.force.com/lightning/r/AuthorizationForm/0cI3F0000008RM6UAM/view`, the ID would be `0cI3F0000008RM6UAM`.
```

Comment:
Hmm, how did this change get in here? I think still needs to say `AuthorizationFormText`

Reply:
Yes that is strange.  I've reverted that change and pushed again.   There is still a diff - cuz of a stupid space.  But I hope your willing to see beyond that.

Reply:
A _space_?! Unforgivable.